erpnext/subcontracting/doctype/subcontracting_order/subcontracting_order.py

```diff
@@ -234,3 +234,11 @@ def update_subcontracting_order_status(sco):
 		sco = frappe.get_doc("Subcontracting Order", sco)
 
 	sco.update_status()
+
+
+@frappe.whitelist()
+def get_rm_valuation_rate(rm_item_code):
+
+	from erpnext.manufacturing.doctype.bom.bom import get_valuation_rate
+
+	return get_valuation_rate({"item_code": rm_item_code})
```

Comment:
This should happen on the server-wide as well, when the document is saved

Reply:
Yes this happens when the document is saved via subcontracting_controller.

Reply:
Oh awesome